src/main/resources/roleconfig/role_sscs.json

```diff
@@ -333,9 +333,9 @@
     ]
   },
   {
-    "name": "superuser",
-    "label": "Superuser",
-    "description": "Superuser role for admin users",
+    "name": "clerk",
```

Comment:
I think this clerk is already present in [role_common.json#L2669](https://github.com/hmcts/am-role-assignment-service/blob/SSCS_WA_1_0/src/main/resources/roleconfig/role_common.json#L2669) for both "SSCS" & "EMPLOYMENT".

Reply:
Correct, I have dropped clerk from role_scss. superuser is also not needed for sscs. See 777f518